homeassistant/components/neato/sensor.py

```diff
@@ -16,10 +24,12 @@
 BATTERY = "Battery"
 
 
-async def async_setup_entry(hass, entry, async_add_entities):
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
     """Set up the Neato sensor using config entry."""
     dev = []
-    neato = hass.data.get(NEATO_LOGIN)
+    neato: NeatoHub | None = hass.data.get(NEATO_LOGIN)
```

Comment:
When can the hub be None?

Reply:
True, based the assumption because of the get()